### DISCUSSION TITLE
What is the relation of Quasar to JustPy and NiceGUI

### DISCUSSION THREAD

**Discussion by metaperl** [zauberzeug/nicegui#134] [Q&A] [answered]
I read [this comment](https://github.com/justpy-org/justpy/issues/404#issuecomment-1214338062) and am wondering how Quasar integrates with:
1. JustPy
2. NiceGUI

does it come out of the box with one or the other?

**Comment by falkoschindler** [comment]
Yes, JustPy comes with (optional) support Quasar. See https://justpy.io/quasar_tutorial/introduction/ for more information.

Because NiceGUI (currently) builds upon JustPy, Quasar is also included. It proved to be a very useful and convenient framework to build web UIs, so it most certainly will remain an essential part of NiceGUI.

**Comment by falkoschindler** [comment, marked as answer]
Since version 1.0 NiceGUI doesn't use JustPy anymore. Quasar, however, is still a crucial building block.
